lib/more_core_extensions/core_ext/string/iec60027_2.rb

```diff
@@ -10,7 +10,7 @@ def iec_60027_2_to_i
       if suffix_index.nil?
         Integer(self)
       else
-        Integer(self[0..-3]) * (2**10)**(suffix_index + 1)
+        Integer(Float(self[0..-3]) * ((2**10)**(suffix_index + 1)))
```

Comment:
My only concern here is that we might hit some weird floating point issue.  I'm wondering if we should do something like:

```ruby
      if suffix_index.nil?
        Integer(self)
      elsif include?(".")
        Integer(Float(self[0..-3]) * (2**10)**(suffix_index + 1))
      else
        Integer(self[0..-3] * ((2**10)**(suffix_index + 1)))
      end
```

I'm sure there's a cleaner way to code that, but it's just for demonstration.